webui/react/package.json

```diff
@@ -36,6 +37,7 @@
     "query-string": "^6.13.5",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
+    "react-helmet": "^6.1.0",
```

Comment:
**chore:** Since this is a public facing library, can you add an attribution license file for `react-helmet` in `tools/scripts/licenses/`? [Here is an example for `antd`](https://github.com/determined-ai/determined/pull/1493/files#diff-5c39a2245e81adb3c3ecad4380aa4bd338839a839a8eb210c79071cbe8fb1379)